fe/src/project/AspNetCore/areas/News/components/NewsItemDetails/NewsItemDetails.js

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import PropTypes from "prop-types";
+import LazyLoad from "react-lazyload";
+import { CalendarToday, PictureAsPdf, Folder, Attachment } from "@material-ui/icons";
+import LazyLoadConstants from "../../../../../../shared/constants/LazyLoadConstants";
+import ResponsiveImage from "../../../../../../shared/components/Picture/ResponsiveImage";
+import NavigationHelper from "../../../../../../shared/helpers/globals/NavigationHelper";
+import { Hash } from "react-feather"
+import {marked} from "marked";
+import moment from "moment";
+
+const NewsItemDetails = (props) => {
+    const handleDownloadFile = (file) => {
+        NavigationHelper.redirect(file.url)
+    }
+
+    return (
+        <div className="section news-details">
+            <div className="container">
+                <div className="columns is-centered">
+                    <div className="column is-8">
+                        <div className="news-details__head">
+                            <h1 className="headline-1">{props.title}</h1>
```

Comment:
**suggestion:** please use title and subtitle of Bulma instead of creating your own styles for it: https://bulma.io/documentation/elements/title/